app/controllers/application_drafts_controller.rb

```diff
@@ -81,8 +81,12 @@ def application_draft
   end
 
   def application_draft_params
-    params.require(:application_draft).
-      permit(:project1_id, :project2_id, :project_plan, :misc_info, :heard_about_it, :voluntary, :voluntary_hours_per_week, :working_together, :why_selected_project)
+    params.require(:application_draft).permit(
+      :project1_id, :project2_id,
+      :plan_project1, :plan_project2,
+      :why_selected_project1, :why_selected_project2,
+      :misc_info, :heard_about_it, :voluntary, :voluntary_hours_per_week, :working_together
```

Comment:
Missing field that are part of the form? That should probably not be the case 😉 
I'll have a look at the list again, should not miss any fields I think, thanx!